### ISSUE TITLE
Mistake when calculating voltage drop

### ISSUE THREAD

**Issue by Da-Technomancer** [Electrical-Age/ElectricalAge#363]
After doing extensive testing with the mod (Note I did not look at the actual code), there seem to be 2 inaccuracies when calculating voltage loss, when using the data on the mod's wiki, wikipedia, and several other sources for comparison as to what it is supposed to be. When testing with light bulbs, medium voltage wire, and the power source, I found two inaccuracies:

A) The resistance of the light bulb in the socket is used to calculate the current, but not the voltage and power loss.
B) When measuring voltage drop between any two points, the voltage drop seems to equal Initial voltage (of the source) x resistance between the two points, all divided by 150. So,
VR/150 = Voltage Loss. On every source I found, it says Power loss is current squared X resistance, or Voltage loss times current, which would mean voltage loss = current X resistance. Clearly, Current X resistance is not the same as VR/150. 

I should also mention that I am not an electrician, so I might have misunderstood what is was intended to be, in which case, feel free to correct me.

**Comment by cm0x4D** [comment]
> A) The resistance of the light bulb in the socket is used to calculate the current, but not the voltage and power loss.

No, it is correct. Take the following example:
(200 Voltage Source)--(10 x Medium Cable)--(200V Bulb)--(GND)
To calculate the actual current, we need the total resistance between the voltage source and GND. This would be 10 times the resistance of the medium cable per block (100mOhm) plus the resistance of the light bulb (1.33 kOhm). 

`Rtot = 1330 Ohm + 10 * 0.1 Ohm = 1331 Ohm`

The current would be `200 V / 1331 Ohm = 0.15 A` which is exactly what we get in the actual game. If we would like to calculate the voltage drop per cable block, it would be the actual current times the resistance of the cable per block: `0.1 Ohm * 0.15 A = 0.015 V`. For the 10 blocks in the example, the voltage drop would be 0.15 V total. So the voltage at the lamp would be in this example 199.85 V, which will be rounded to 200V when displaying the value.

> B) When measuring voltage drop between any two points, the voltage drop seems to equal Initial voltage (of the source) x resistance between the two points, all divided by 150. So,
> VR/150 = Voltage Loss. On every source I found, it says Power loss is current squared X resistance, or Voltage loss times current, which would mean voltage loss = current X resistance. Clearly, Current X resistance is not the same as VR/150.

No, that is not the case, the voltage drop is clearly the current times the resistance.

Maybe you messed up the units of the resistances, mOhm it milli-Ohms (\* 10^-3) and kOhm is kilo-Ohms (\* 10^3)
